datafusion/sql/src/planner.rs

```diff
@@ -6239,11 +6256,7 @@ mod tests {
         // param is not number following the $ sign
         // panic due to error returned from the parser
         let sql = "PREPARE my_plan(INT) AS SELECT id, age  FROM person WHERE age = $foo";
-
-        let expected_plan = "whatever";
-        let expected_dt = "whatever";
-
-        prepare_stmt_quick_test(sql, expected_plan, expected_dt);
+        logical_plan(sql).unwrap();
```

Comment:
Instead of calling `prepare_stmt_quick_test` that will first generate logical plan (and then panic for this test), I call `logical_plan` directly to simplify the test but still test the same things